scst-scan/creating-scan-policy-template.hbs.md

```diff
@@ -0,0 +1,262 @@
+# Enabling policy enforcement with Scan 2.0 in a supply chain
+
+Policy enforcement is not inbuilt in Scan 2.0. It can be achieved by the creating a 
+`ClusterImageTemplate` that stamps out a Tekton `TaskRun` that evaluates the vulnerabilities
+and enforces the policy set. A sample of the task run along with a the cluster image template
```

Comment:
> task run
Nit: Should we say "TaskRun" to be more specific so that it is clear it is a Tekton TaskRun?

> cluster image template
Nit: ClusterImageTemplate

Reply:
will update it thabnks!